src/System.Private.ServiceModel/tests/Common/Scenarios/ScenarioTestTypes.cs

```diff
@@ -1230,3 +1231,18 @@ public class UniqueType
 {
     public string stringValue;
 }
+    
+[XmlType(Namespace="urn:TestWebServices/MyWebService/")]
+public class TestMessageHeader {        
+    private string message;
```

Comment:
Nit: we should probably choose a name and include a comment that makes it very clear that this type must not be used by any other service or operation or [ServiceKnownType].  Otherwise someone might reuse this type in another test and accidentally tell SG to make a DataContract serializer for it, which would defeat what this test is currently testing.

@StephenBonikowsky might have recommendations because he has other types like this that say "no one else can use me".